apps/api/src/common/hotcache.ts

```diff
@@ -0,0 +1,22 @@
+class CacheObject {
+    constructor(public value: any, public expiration: number) {}
+    isExpired() {
+        return this.expiration < Date.now()
+    }
+}
+
+export class HotCache {
+    private cache: { [key: string]: CacheObject } = {}
+
+    get(key: string) {
+        const cacheObject = this.cache[key]
+        if (cacheObject && !cacheObject.isExpired()) {
```

Comment:
We might want a separate condition to check if cacheObject.isExpired(), and if it is we should also 
`delete cacheObject`

Reply:
Don't see a reason to delete anything as of now.
This would only complicate things imho.